src/timers/RedisRetriesRepublishTimerTask.js

```diff
@@ -31,7 +31,7 @@ class RedisRetriesRepublishTimerTask extends SkipTimer {
       return;
     }
 
-    // Pressess each message.
+    // Presses each message.
```

Comment:
Should this be process instead of presses?

```suggestion
    // Process each message.
```